phoenix-core/src/it/java/org/apache/phoenix/end2end/Dummy.java

```diff
@@ -1,3 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
```

Comment:
This entire file can be removed; I believe it's been causing ASF license failures and it doesn't do anything. I think a previous JIRA checked it in by mistake.

Reply:
thank you , I will remove this class.

Reply:
@gjacoby126 ， I remove this class , please review again. thank you.

Reply:
PHOENIX-6272 is tracking this?

Reply:
oh, thank you  @virajjasani ,I will assign me to PHOENIX-6272, and remove Dummy.java in this PR.